src/BarChart/RenderStackBars.tsx

```diff
@@ -77,7 +77,12 @@ const RenderStackBars = (props: StackedBarChartPropsType) => {
         style={[
           {
             width:
-              (item.stacks[0].barWidth || props.barWidth || 30) + spacing / 2,
+              (item.labelWidth ||
+                props.labelWidth ||
```

Comment:
This won't work until https://github.com/Abhinandan-Kushwaha/gifted-charts-core/pull/36 is merged